tests/Feature/Pages/UpdatePageTest.php

```diff
@@ -132,16 +132,20 @@ public function slugcheck_takes_archived_into_account_as_well()
     /** @test */
     public function only_nl_is_required()
```

Comment:
When an entire translation is empty, it should still be updated as it removes a translation.